fr/news/_posts/2015-03-06-google-summer-of-code-2015.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: news_post
+title: "Google Summer of Code 2015"
+author: "Federico Builes"
+translator: "Jean-Denis Vauguet"
+date: 2015-03-06 10:48:37 +0000
+lang: fr
+---
+
+Ruby va participer au [Google Summer of Code 2015][gsoc] en tant que structure
+encadrante, pour chapeauter tous les projets liés à [Ruby lui-même][ruby-ideas]
+mais aussi [JRuby][jruy-ideas], [Celluloid][celluloid] et [autres][ideas]. Les
+candidatures d'étudiants sont acceptées à partir du 16 mars et ce jusqu'au 27
+mars ([plus de détails ici][timeline]).
+
+Si participer vous intéresse, que ce soit en qualité d'étudiant ou de mentor,
+il vous suffit de rejoindre notre [liste de diffusion][ml] pour en savoir plus.
+Vous pouver par ailleurs consulter une liste d'idées de projets sur le
+[wiki RubyGSoC][ideas].
+
+[Ruby on Rails][ror] et [SciRuby][sciruby] seront aussi de la partie en tant
+que structures encadrantes pour cette édition 2015. Si vous avez une idée
+plus spécifiquement liée à l'un de ces deux univers, consultez plutôt leurs
+communiqués respectifs : [RoR][ror-announcement], [SciRuby][sciruby-ideas].
```

Comment:
I think it would be better if use `[Ruby on Rails][ror-announcement]` instead of `[RoR][ror-announcement]`.